libtiledbvcf/src/CMakeLists.txt

```diff
@@ -291,52 +245,85 @@ set_target_properties(tiledbvcf
 include(GNUInstallDirs)
 
 # Set rpath to be relative to the .so.
-if (APPLE)
-  set_target_properties(tiledbvcf PROPERTIES INSTALL_RPATH "@loader_path/")
-else()
-  set_target_properties(tiledbvcf PROPERTIES INSTALL_RPATH "$ORIGIN/")
-  if(WIN32)
-    set_target_properties(tiledbvcf4test PROPERTIES INSTALL_RPATH "$ORIGIN/")
-  endif()
+#if (APPLE)
+#  set_target_properties(tiledbvcf PROPERTIES INSTALL_RPATH "@loader_path/")
+#else()
+#  set_target_properties(tiledbvcf PROPERTIES INSTALL_RPATH "$ORIGIN/")
+#endif()
+
+#set_property(
+#        TARGET tiledbvcf-bin
+#        PROPERTY INSTALL_RPATH
+#        "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+#        "${CMAKE_INSTALL_PREFIX}/lib"
+#)
+
+#if(WIN32)
+#  # Having the .dll and the .exe with the same name was resulting in a .exp
+#  # file for the .exe being 'last out' (replacing one of same name for the .dll)
+#  # and was causing the python api extension
+#  # to link to the .exe, which was not functional.
+#  # So, tiledbvcf-bin now sets an output (above somewhere) of tiledbvcfcli.exe
+#  # and we install it here RENAMEing it to match the known name in pre-existing
+#  # *nix world.
+#  install(PROGRAMS $<TARGET_FILE:tiledbvcf-bin>
+#          RENAME tiledbvcf.exe
+#          DESTINATION tiledbvcf
+#  )
+#else()
+#  install(
+#          TARGETS tiledbvcf-bin
+#          RUNTIME DESTINATION tiledbvcf
+#  )
+#endif()
+
+if(TILEDBVCF_INSTALL_TILEDB)
+  install(
+          IMPORTED_RUNTIME_ARTIFACTS
+          TileDB::tiledb_shared
+          RUNTIME DESTINATION tiledbvcf
+          LIBRARY DESTINATION tiledbvcf/lib
```

Comment:
We should be using `bin` and `lib`.

Reply:
`CMAKE_INSTALL_{BINDIR,LIBDIR}` for full windows compatibility? https://github.com/TileDB-Inc/TileDB/blob/738f238d99776e03d2eb36ddf0cd55254f9ba32f/tiledb/CMakeLists.txt#L852

Reply:
Definitely, after the SEGFAULT is fixed I'll clean up the changes to be more organized and use CMAKE_* variables

Reply:
This is kind of an issue, since `CMAKE_INSTALL_LIBDIR` points to `site-packages/lib` which is not correct for our case. Looking at other projects they also use just folder names: https://github.com/ssciwr/clang-format-wheel/blob/dc29f5e2a074dee8e862eb29fd0de1d84f41707a/CMakeLists.txt#L43

I don't think that we need to use `GNUInstallDirs` CMake module. However, perhaps this will come into play when I'll be testing Conda packaging.